internals/models.py

```diff
@@ -732,12 +745,12 @@ def get_all(self, limit=None, order='-updated', filterby=None,
     feature_list = ramcache.get(KEY)
 
     if feature_list is None or update_cache:
-      query = Feature.all().order(order) #.order('name')
-      query.filter('deleted =', False)
+      query = Feature.query().order(-Feature.updated) #.order('name')
```

Comment:
I migrated line 748 incorrectly earlier. `order` is not a property of Feature. Instead, it is a parameter of the function which contain these lines. On investigating, I found that `order` will always have the value `'-updated'`. Hence, I have changed it to `-Feature.updated`

Reply:
That seems good.